### PR TITLE
publish to github not docker

### PR DESCRIPTION
**Pull request by cozmo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,8 +14,7 @@ $(MOCKGEN):
 	go get -u github.com/golang/mock/mockgen
 
 build: hardcoded/hardcoded.go
-	# disable CGO and link completely statically (this is to enable us to run in containers that don't use glibc)
-	CGO_ENABLED=0 go build -installsuffix cgo -o bin/wag
+	go build -installsuffix cgo -o bin/wag
```
Comment (rgarcia) [inline Makefile]: `-installsuffix cgo` shouldn't be required anymore

**Comment by rgarcia** [comment]
lgtm

**Comment by kvigen** [comment]
i assume we did this because we don't need java anymore so the docker container wasn't adding much value?

**Comment by cozmo** [comment]
Yep

**Pull request merged**